docs/running-on-mesos.md

```diff
@@ -613,6 +621,41 @@ See the [configuration page](configuration.html) for information on Spark config
     driver disconnects, the master immediately tears down the framework.
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.rejectOfferDuration</code></td>
+  <td><code>120s</code></td>
+  <td>
+    The amount of time that the master will reject offer after declining 
```

Comment:
This doesn't sound correct. The mesos.proto (https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L2310) states:
```
Time to consider unused resources refused. Note that all unused
resources will be considered refused and use the default value
(below) regardless of whether Filters was passed to
SchedulerDriver::launchTasks. You MUST pass Filters with this
field set to change this behavior (i.e., get another offer which
includes unused resources sooner or later than the default).
``` 
some simple word-smithing or a link should make it clearer.

Reply:
Thanks for the comment, I will update it.